pr_agent/algo/utils.py

```diff
@@ -1125,8 +1125,8 @@ def get_version() -> str:
                     return data["project"]["version"]
                 else:
                     get_logger().warning("Version not found in pyproject.toml")
-        else:
-            get_logger().warning("Unable to determine local version from pyproject.toml")
+    else:
+        get_logger().warning("Unable to determine local version from pyproject.toml")
```

Comment:
i am not sure i agree with this change.

If I have a pip package, than I won't find a "pyproject.toml" file. but its ok. it does not justify a "warning" message.

Reply:
That's a good point, this is the old code. You can see this code is not reachable
![CleanShot 2024-12-23 at 18 58 26](https://github.com/user-attachments/assets/1de28aff-7ba4-4125-a41d-be757e42b936)
And it doesn't have the same code block with `if os.path.exists("pyproject.toml"):`

Reply:
![image](https://github.com/user-attachments/assets/c91e8d4e-ec91-4bbe-aca1-532f45e429a6)

maybe its due to your python version ?

Reply:
I'm not sure, I'm using 3.13. Let me check it with 3.12 and get back to you afterward.

Reply:
i am saying i dont see unreachable code warning on my platform.

And I am not sure from the code why it should be unreachable